sql/testdata/churn_sql.go

```diff
@@ -1,3 +1,20 @@
+package testdata
+
+// Copyright 2019 The SQLFlow Authors. All rights reserved.
```

Comment:
Copyright should come before `package testdata`. Is this a bug in `scripts/copyright.py`?

Reply:
Seems so. Will do more tests and try to fix this this week.